index.html

```diff
@@ -0,0 +1,179 @@
+        <link rel="stylesheet" href="./style/body/navigation/button/button.css"/>
+        <link rel="stylesheet" href="./style/body/navigation/button/_style/button_style_bordered.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/templates-selector.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/__header/templates-selector__header.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/__header/search-box/search-box.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/__header/search-box/__input/search-box__input.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/wrapper/wrapper.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/edit-buttons/edit-buttons.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/template-item/template-item.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/template-item/__text/template-item__text.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/template-item/__counters/template-item__counters.css"/>
+        <link rel="stylesheet" href="./style/body/scroll-panel/scroll-panel.css"/>
+        <link rel="stylesheet" href="./style/footer/footer.css"/>
+    </head>
+    <body class="body">
+        <header class="header">
+            <a class="logo" href=""></a>
+            <ul class="menu">
+                <li>
+                    <a href="#user">
+                        <i class="fa fa-user" aria-hidden="true"></i>
+                        <span>Привет, Админ</span>
+                        <i class="fa fa-caret-down" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li><a href="#about">О компании</a></li>
+            </ul>
+        </header>
+
+        <div class="container">
+            <ul class="navigation">
+                <li><a class="button button_style_bordered" href="#new-interview" >Новый опрос</a></li>
+                <li><a class="button button_style_bordered" href="#my-interviews">Мои опросы</a></li>
+                <li><a class="button button_style_bordered" href="#interview-templates">Шаблоны опросов</a></li>
+                <li><a class="button button_style_bordered" href="#users">Пользователи </a></li>
+            </ul>
+            <div class="templates-selector">
+                <div class="templates-selector__header">
+                    <h2 class="templates-selector__text">Шаблоны</h2>
+                    <a class="button button_style_bordered" href="#new-template">Новый шаблон</a>
+                    <div class="search-box">
+                        <i class="fa fa-search"></i>
+                        <input class="search-box__input" type="search" placeholder="Search..." />
+                    </div>
+                </div>
+
+                <div class="wrapper" tabindex="1">
```

Comment:
When the title has a lot of text, template is broken:  https://www.screencast.com/t/Pb7zFR9NCFhk 
Please consider using "text-overflow: ellipsis;" for the title.